draft-irtf-cfrg-opaque.md

```diff
@@ -624,45 +612,31 @@ Output:
 
 Steps:
 1. N = Unblind(blind, response.data)
-2. y = Finalize(pwdU, N, "OPAQUE00")
+2. y = Finalize(pwdU, N, "OPAQUE01")
 3. rwdU = HKDF-Extract("rwdU", Harden(y, params))
-4. Create secret_credentials with CredentialExtensions matching that
-   contained in response.secret_credentials_list
-5. Create cleartext_credentials with CredentialExtensions matching that
-   contained in response.cleartext_credentials_list
-6. pt = SerializeExtensions(secret_credentials)
+4. Create secret_credentials with credentials matching those needed to
+   construct the `SecretCredentials` structure
+5. Create cleartext_credentials with credentials matching those needed to
+   construct the `CleartextCredentials` structure
+6. pt = Serialize(secret_credentials)
```

Comment:
If SecretCredentials is always equal to the client secret, wouldn't it be more straightforward to put `pt = Serialize(secret)` ?

Reply:
I think it may be beneficial to keep this level of generality here in case we add support for other modes that set SecretCredentials to something else in the future. This will hopefully also coincide with the changes that will be made in #105 which will pass in a generic credentials parameter.